3term-study/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+  let answer = [];
+  n = String(n);
+  for (let num of n) {
+    answer.push(Number(num));
+  }
+  return answer.reverse();
```

Comment:
```suggestion
    answer.unshift(Number(num));
  }
  return answer;
```
배열에 push (뒤에 push) 후 reverse() 하는 것 보다 unshift (앞에 push) 하는 것이 더 편할 것 같습니다.